terraform/common/iam.tf

```diff
@@ -89,14 +89,40 @@ resource aws_iam_user_policy_attachment cloudwatch {
   policy_arn = aws_iam_policy.cloudwatch.arn
 }
 
+# ACM
+
+data aws_iam_policy_document acm {
+  statement {
+    actions   = ["acm:DescribeCertificate"]
```

Comment:
Can you put the 3 acm actions in the same list? This would be more consistent with the rest of the file.

Reply:
Thanks. I've grouped as per S3 statements.